src/test/java/com/auth0/json/mgmt/guardian/FactorTest.java

```diff
@@ -28,15 +27,7 @@ public void shouldDeserialize() throws Exception {
 
         assertThat(factor, is(notNullValue()));
         assertThat(factor.isEnabled(), is(true));
+        assertThat(factor.getName(), is("sms"));
```

Comment:
that's exactly why I removed the setters.
this is the test for <deserialize>, there it will have all values